kalite/settings/base.py

```diff
@@ -322,6 +323,10 @@
 
 ROOT_URLCONF = "kalite.distributed.urls"
 
+backup_dirpath = expanduser("~")
+backup_dirpath = os.path.join(backup_dirpath, 'ka-lite-backups')
+DBBACKUP_BACKUP_DIRECTORY = backup_dirpath
```

Comment:
Oh, you keep redefining BACKUP_DIRPATH. Why not just use the one in settings? You can then use the value using:

```python
from django.conf import settings

settings.BACKUP_DIRPATH
```